src/amo/components/App/styles.scss

```diff
@@ -31,6 +31,9 @@ textarea {
 *::before,
 *::after {
   box-sizing: inherit;
+  // This is needed for Safari, which does not support `anywhere`.
+
```

Comment:
```suggestion
```

Reply:
heh wait

Reply:
i followed the comment below that has also one line space :P

Reply:
whaaaat?

Reply:
ah I see now :(